system/jhtree/jhtree.cpp

```diff
@@ -2088,7 +2088,7 @@ void IndexRowFilter::finish(size32_t _keyedSize)
 
 void IndexRowFilter::describe(StringBuffer &out) const
 {
-    for (unsigned idx=0; idx < lastRealSeg(); idx++)
+    for (unsigned idx=0; idx <= lastRealSeg(); idx++)
```

Comment:
This is a separate bug spotted during testing

Reply:
similar problem if there are no filters.